main_test.go

```diff
@@ -204,5 +204,30 @@ var _ = Describe("nfsbroker Main", func() {
 				Expect(catalog.Services[0].Plans[0].Description).To(Equal("A preexisting filesystem"))
 			})
 		})
+
+		Context("given config argument", func() {
+			BeforeEach(func() {
+				args = append(args, "-sourceAllowed", "uid,gid", "-mountAllowed", "nfs_uid,nfs_gid")
```

Comment:
consider removing the test, not real added value, sice the args parsing is in the args library